hermes/crates/cbork/cddl-parser/src/lib.rs

```diff
@@ -7,20 +7,53 @@
 use std::fmt::Debug;
 
 pub use pest::Parser;
-use pest_derive::Parser;
 
 extern crate derive_more;
 use derive_more::{Display, From};
+use pest::error::Error;
 
-// Parser with DEBUG rules.  These rules are only used in tests.
-#[derive(Parser)]
-#[grammar = "grammar/cddl.pest"]
-#[grammar = "grammar/cddl_test.pest"] // Ideally this would only be used in tests.
-pub struct CDDLParser;
+pub mod rfc_8610 {
+    pub use pest::Parser;
+
+    #[derive(pest_derive::Parser)]
+    #[grammar = "grammar/cddl.pest"]
+    pub struct RFC8610Parser;
+}
+
```

Comment:
Add the other `.pest` files that are planned, and just have them with a comment at the top explaining what they will contain.

Then add the other 2 parser modules we plan here, so that in future all we need to do is add the grammar and tests and its done.